frontend/src/main/scala/bloop/engine/tasks/Tasks.scala

```diff
@@ -212,7 +212,8 @@ object Tasks {
     val loader = ClasspathUtilities.makeLoader(classpathFiles, scalaInstance)
     val compiler = state.compilerCache.get(scalaInstance).scalac.asInstanceOf[AnalyzingCompiler]
     val classpathOptions = ClasspathOptionsUtil.repl
-    compiler.console(classpathFiles, project.scalacOptions, classpathOptions, "", "", state.logger)(
+    val options = project.scalacOptions :+ "-Xnojline"
```

Comment:
I'd love to use a constant here, but `scala.tools.nsc.settings.ScalaSettings` has no companion object and I don't think that creating our own instance just to extract this value makes much sense.

Reply:
The flag is unlikely to change, so I don't think it's a big deal :+1: